PackageInfo.g

```diff
@@ -113,7 +113,7 @@ PackageDoc := rec(
 
 
 Dependencies := rec(
-  GAP := "4.9.0",
+  GAP := "4.10.0",
```

Comment:
Perhaps the commit doing this could briefly state why it is necessary?

Reply:
Will do. Tests fail under 4.9, and because 4.10.0 is already one and a half years old, I think it's fine to require it at least.